src/v35.js

```diff
@@ -1,12 +1,53 @@
 import bytesToUuid from './bytesToUuid.js';
 
+function hexSymToDecNum(n) {
```

Comment:
Is this so much faster than `parseInt(n, 16)`?

Reply:
Yes, faster than `parseInt(uuid.substr(i, 2), 16)`
You can check it yourself.

Reply:
This is ridiculous 😂 🤷‍♂️

Reply:
Noticeably faster, otherwise I would have done through `parseInt` %)

Reply:
Don't get me wrong: I didn't mean to question, that your solution is considerably faster, I was just amazed by the fact…